repositories.md

```diff
@@ -5,9 +5,75 @@ css_id: repositories
+## Secure Source of Time
+
+Without access to a secure source of time, ECUs may be prevented from receiving the most recent updates. If the ECU's time is too high, the ECU will detect that the current valid metadata is expired and will be unable to perform an update. If the ECU's time is too low, an attacker can freeze or replay old metadata to the ECU.  (ECUs in Uptane will not accept an earlier time than what has been seen before signed with the same key.)
+
+To prevent these issues, ECUs need access to a secure source of time. If an ECU does not have a secure clock, we recommend the use of a Time Server for time attestations. This section describes how a Time Server can be used in an Uptane implementation.
+
+### Time server
+
+A Time Server is a server that is responsible for providing a secure source of time.
+
+The Time Server exists to inform ECUs about the current time in a cryptographically secure way, since many ECUs in a vehicle do not have a reliable source of time. The Time Server receives a list of tokens from vehicles, and returns back a list of signed records containing every token in the original list of tokens received and at least one instance of the current time.
+
+If the Time Server is used, it is CONDITIONALLY REQUIRED to conform to the following requirements:
+
+* When the Time Server receives a sequence of tokens from a vehicle, it will provide one or more signed responses, containing the time along with these tokens. It MAY produce either one signed time attestation containing the current time and all tokens, or multiple time attestations each containing the current time and one or more tokens. All tokens should be included in the response.
+
+* The Time Server will expose a public interface allowing primaries to communicate with it. This communication MAY occur over FTP, FTPS, SFTP, HTTP, HTTPS, or another transport control of the implementor's choice.
+
+* Rotation of the the Time Server's key is performed by listing the new key in the Director's Root metadata, in the same manner as other role keys are listed, and also in the custom field of the Director repository's Targets metadata (for partial verification Secondaries).
+
+#### Changes to the Director repository
+If a Time Server is in use, a representation of the Time Server public key is CONDITIONALLY REQUIRED in Director repository root metadata.
+
+If a Time Server is implemented AND partial verification Secondaries are used, the following metadata is CONDITIONALLY REQUIRED in the Director repository's Targets metadata:
+
+* A representation of the public key(s) for the Time Server, similar to the representation of public keys in Root metadata.
+
+Listing the public key of the Time Server in Director Targets metadata is necessary to allow partial-verification Secondaries to perform Time Server key rotation.
```

Comment:
Sorry, one more!
```suggestion
Listing the public key of the Time Server in Director Targets metadata is necessary to allow partial verification Secondaries to perform Time Server key rotation.
```

Reply:
I would suggest, if this PR is now technically correct that we commit what is here. I think the comment of @tkfu needs to be addressed across the board on all pages. So rather than hold up this already lengthy set of commits, let's merge this PR and address the style issue at a later time. I would like to do such a pass myself once the content is more or less set.